data/index.php

```diff
@@ -105,12 +111,16 @@ function ($matches) {
 			case 'sensors':
 				include ("../connect.php");
 				$database = Connection();
+				//Build where clause
 				$WHERE = "";
 				if ($start) $WHERE.= " WHERE timestamp >= '" . $database->real_escape_string($start) . "'";
 				if ($end) $WHERE.= ($WHERE?" AND ":" WHERE ")."timestamp <= '" . $database->real_escape_string($end) . "'";
 				if ($ids) $WHERE.= ($WHERE?" AND ":" WHERE ")."station_id IN (" . $database->real_escape_string($ids) . ")";
-				$SORT = ' ORDER BY timestamp ASC';
-				$query = "SELECT * FROM sensors_measurement".$WHERE.$SORT;
+				//Determine sort order
+				$SORT = ' ORDER BY timestamp '.$_GET['sort_order'];
```

Comment:
I believe this needs some validation, including sort_order in the query directly leaves SQL injection possible. Probably just checking with `in_array` against `['asc', 'desc']`, maybe doing a `strtolower` first to make it case insensitive?